test/tests/functional/pbs_hooksmoketest.py

```diff
@@ -84,31 +80,30 @@ def clean_hooks(self):
             self.server.manager(MGR_CMD_LIST, HOOK)
         except:
             pass
-        if 'my_hook_queuejob' in self.server.hooks:
-            self.server.manager(MGR_CMD_DELETE, HOOK, id="my_hook_queuejob")
-        if 'my_hook_queuejob2' in self.server.hooks:
-            self.server.manager(MGR_CMD_DELETE, HOOK, id="my_hook_queuejob2")
-        if 'my_hook' in self.server.hooks:
-            self.server.manager(MGR_CMD_DELETE, HOOK, id="my_hook")
+        del_hook = ['test_hook_queuejob',
+                    'test_hook_queuejob2', 'test_hook2', 'test_hook']
+        for i in del_hook:
+            if i in self.server.hooks:
+                self.server.manager(MGR_CMD_DELETE, HOOK, id=i)
```

Comment:
this is unnecessary. we are looping through the list just to find the one hook that was created in a test.
We can use only one name, and delete that hook in tearDown()